pkg/client/common/discovery/dynamicdiscovery/service_test.go

```diff
@@ -0,0 +1,91 @@
+// +build testing
```

Comment:
I'm surprised to see a build tag in a *_test.go file.

Reply:
https://github.com/hyperledger/fabric-sdk-go/blob/main/pkg/client/common/selection/fabricselection/selection_test.go#L1

hm, I saw it in all _test.go files in that pkg